### PR TITLE
changed pre-release AppVeyor builds to end in -b123

### PR DESCRIPTION
**Pull request by ctaggart**
added into to informational version,
updated readme with AppVeyor NuGet feed info

### CODE REVIEW

**Comment by ctaggart** [comment]
ok, @dsyme, this looks good now. The pre-release build versions will be based on whatever you have in RELEASE_NOTES.md. 3 digits is the semver way, but NuGet supports 4 if you wish. If you merge these changes into your 4.0 branch, just update the RELEASE_NOTES.md appropriately there. Please update the NuGet feed name to match what I've documented.

https://ci.appveyor.com/project/fsgit/fsharp-compiler-service/history

image

File: README.md [hunk 1]
```diff
@@ -22,6 +21,32 @@ Head (branch ``master``), Mono 3.x, OSX + unit tests (Travis) [![Build Status](h
 
 Head (branch ``master``), Windows Server 2012 R2 + unit tests (AppVeyor)  [![Build status](https://ci.appveyor.com/api/projects/status/3yllu2qh19brk61d)](https://ci.appveyor.com/project/fsgit/fsharp-compiler-service)
 
+NuGet Feed
+------------
+
+Stable builds are available in the NuGet Gallery:
+[https://www.nuget.org/packages/FSharp.Compiler.Service](https://www.nuget.org/packages/FSharp.Compiler.Service)
+
+All AppVeyor builds are available using the NuGet feed: https://ci.appveyor.com/nuget/fsgit-fsharp
```
Comment (dsyme) [inline README.md]: THis should be https://ci.appveyor.com/nuget/fsgit-fsharp-compiler-service

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
-#### 1.3.1.0 -
+#### 1.3.2 -
```
Comment (dsyme) [inline RELEASE_NOTES.md]: The "3.1" was intended to refer to the language version supported (1.3.1.x v. 1.4.0.x) . Not exactly SemVer I know. Can you use 1.3.1.1 or else suggest a different version numbering stream for the 3.1 and 4.0 versions? thx

Comment (ctaggart) [inline RELEASE_NOTES.md]: You have 4 number to work with in a version. You could match the first two to the product version of `3.1` and `4.0`. The 3 number would be the number that you increment for stable releases.

Comment (dsyme) [inline RELEASE_NOTES.md]: I'm not really sure. I'm not fussed about following SemVer yet here - we haven't done that so far and no reason to start just yet I think.

I was thinking we'd use the "1" out the front for very major revisions and the fourth for very minor revisions, But perhaps we should just switch to only publish F# 4.0 support anyway at some point very soon.  In that case 1.3.1.x and 1.4.0.x seem to work well enough for now.

**Pull request merged**